packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -166,6 +166,27 @@ class _CodeViewState extends State<CodeView>
 
   @override
   Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+        valueListenable: widget.controller.showFileOpener,
+        builder: (context, showFileOpener, _) {
+          return Stack(
+            children: [
+              buildCodeviewContent(context),
+              if (showFileOpener)
+                Positioned(
+                  left: 110.0,
```

Comment:
prefer consts with a descriptive name over "magic" numbers. For example, see another const in this class `searchFieldRightPadding`

Reply:
Done, thanks!